xm-commons-lep/src/test/java/com/icthh/xm/commons/lep/TenantScriptStorageUnitTest.java

```diff
@@ -0,0 +1,56 @@
+package com.icthh.xm.commons.lep;
+
+import static com.icthh.xm.commons.lep.TenantScriptStorage.CLASSPATH;
+import static com.icthh.xm.commons.lep.TenantScriptStorage.FILE;
+import static com.icthh.xm.commons.lep.TenantScriptStorage.XM_MS_CONFIG;
+import static org.apache.commons.io.FilenameUtils.separatorsToSystem;
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class TenantScriptStorageUnitTest {
+
+    private static final String TENANT_KEY = "xm";
+    private static final String APP_NAME = "activation";
+    private static final String PATH = "/commons/functions/aggregation";
+    private static final String URL_PREFIX_COMMONS_ENVIRONMENT = "/commons/environment";
+    private static final String URL_PREFIX_COMMONS_TENANT = "/commons/tenant";
+
+    @Test
+    public void testResolveLepForClassPathScriptStorage() {
+        String envLepPath = CLASSPATH.resolvePath(TENANT_KEY, APP_NAME, URL_PREFIX_COMMONS_ENVIRONMENT + PATH);
+        String tenantLepPath = CLASSPATH.resolvePath(TENANT_KEY, APP_NAME, URL_PREFIX_COMMONS_TENANT + PATH);
+        String localLepPath = CLASSPATH.resolvePath(TENANT_KEY, APP_NAME, PATH);
+
+        assertEquals("classpath:/lep/custom/commons/functions/aggregation", envLepPath);
+        assertEquals("classpath:/lep/custom/xm/commons/functions/aggregation", tenantLepPath);
+        assertEquals("classpath:/lep/custom/xm/commons/functions/aggregation", localLepPath);
```

Comment:
@igordumchykov is it correct that these two lines above are the same? Seems localLepPath should contain the application name.

Reply:
I was wrong. These changes failed tests, fix: https://github.com/xm-online/xm-commons/pull/119